content/release-10-17-0/announcements-10-17-0.md

```diff
@@ -24,6 +24,12 @@ As of release 10.20+, the default value for the `withChildren` parameter of the
 As a consequence, if a request is supposed to return child assets, devices and additions, a parameter `?withChildren=true` has to be provided in the query.
 This change is necessary to improve the Inventory API performance.
 
+##### Breaking change in the Inventory API -- change of the return code for delete of managed object
+
+As of release 10.19+, when delete of managed object and its dependencies doesn't finish right away, the platform will return 202 (Accepted) http status code instead of 204 (No content).
```

Comment:
```suggestion
As of release 10.19+, if the deletion of a managed object and its dependencies doesn´t finish immediately, the platform will return a 202 (Accepted) HTTP status code instead of 204 (No content).
```